SpatialGDK/Documentation/content/get-started/gdk-and-starter-project.md

```diff
@@ -0,0 +1,70 @@
+    * (HTTPS) `git clone https://github.com/spatialos/UnrealGDKStarterProject.git`
+    * (SSH) `git clone git@github.com:spatialos/UnrealGDKStarterProject.git`
+1. Navigate into `<StarterProjectRepoRoot>/Game/Plugins`. You may need to create the Plugins folder if you don't have any plugins installed yet.
+1.  In a Git Bash terminal window, clone the [Unreal GDK](https://github.com/spatialos/UnrealGDK) repository by running either:
+    * (HTTPS) `git clone https://github.com/spatialos/UnrealGDK.git`
+    * (SSH) `git clone git@github.com:spatialos/UnrealGDK.git`
+
+**Note** You need to ensure that the root folder of the Unreal GDK repo is called UnrealGDK (`<StarterProjectRepoRoot>/Game/Plugins/UnrealGDK/...`)
+
+[//]: # (TODO: This whole section below deserves some screenshots to show users what to expect. This was done well on the Unity onobarding docs. Example: https://docs.improbable.io/unity/alpha/content/get-started/get-playing)	
+
+### Building
+
+> If you want to port an existing Unreal project to the Unreal GDK, follow [this guide]({{urlRoot}}/content/porting-unreal-project-to-gdk) instead of continuing to follow these steps.
+
+Build the Unreal GDK module dependencies which the Starter Project needs to work with the GDK and add the Unreal GDK to the Starter Project.
+
+1. Open **File Explorer**, navigate to the root directory of the Unreal GDK repository, and double-click **`Setup.bat`**. You may be prompted to sign into your SpatialOS account if you have not already.
+1. Set the Starter Project to work with the Unreal Engine fork you cloned and installed earlier. To do this:
+    1. In File Explorer, navigate to the root directory of the Unreal GDK Starter Project repository, and then to the **Game** directory within it.
+    1. Right-click **StarterProject.uproject** and select **Switch Unreal Engine version**.
+    1. Select the path to the Unreal Engine fork you cloned earlier.
+1. In the same directory, open **StarterProject.sln** with Visual Studio.
+1. In the Solution Explorer window, right-click on **StarterProject** and select **Build**.
+1. Open **StarterProject.uproject** in the Unreal Editor and click `Schema` to generate schema and then `Snapshot` to generate a snapshot.
+
+### Running the Starter Project locally
+
+1. In the Unreal Editor, on the SpatialOS Unreal GDK toolbar, click **Launch**. Wait until you see the output `SpatialOS ready. Access the inspector at http://localhost:21000/inspector`.
+1. On the Unreal Editor toolbar, open the **Play** drop-down menu.
+1. Under **Multiplayer Options**, enter the number of players as **2** and check the box next to Run **Dedicated Server**. Then, under Modes, select **New Editor Window (PIE)**.
+1. On the toolbar, click **Play** to run the game.
+1. When you're done, click **Stop** to stop the client worker, and click the **Stop** button in the SpatialOS section of the toolbar to stop your local SpatialOS deployment.
```

Comment:
"client-worker" also, we should link to the GDK glossary for "worker" because we have suddenly introduced this term from nowhere. [glossary]({{URLRoot}}/content/glossary#workers)